src/ethernet/RsRtspClient.cpp

```diff
@@ -364,16 +368,16 @@ void updateExtrinsicsMap(rs2_video_stream videoStream, std::string extrinsics_st
 void RsRTSPClient::continueAfterDESCRIBE(RTSPClient *rtspClient, int resultCode, char *resultString)
 {
     std::string resultStr;
-    if(nullptr != resultString)
+    if (nullptr != resultString)
     {
         resultStr = resultString;
-        delete [] resultString;
+        delete[] resultString;
```

Comment:
Is it possible to refactor this so that the input string will be passed as const ref instead of pointers?
This is really not safe and prone to bugs, e.g. what happens if the input string is passed into a separate thread.